playbooks/adhoc/uninstall_openshift.yml

```diff
@@ -393,6 +393,8 @@
     with_subelements:
     - "{{ directories.results | default([]) }}"
     - files
+    # Ignore failure when openshift.local.volumes is mounted.
+    failed_when: false
```

Comment:
@micahel71 were you following the rest of the playbook style?  We should be using `ignore_errors` not `failed_when`.

Reply:
@jcpowermac as I mentioned, I just copied this code from the very same file (look at row 272-277)
```yaml

  - file: path={{ item.1.path }} state=absent
    with_subelements:
    - "{{ directories.results | default([]) }}"
    - files
    # Ignore failure when openshift.local.volumes is mounted.
    failed_when: false
```
As this was existing code, I expected that this would be OK. But if you insists I can fix this at both places.